src/The_Cartridge_Header.md

```diff
@@ -247,7 +247,157 @@ This byte is used in older (pre-SGB) cartridges to specify the game's publisher.
 However, the value `$33` indicates that the [New Licensee Code](<#0144-0145 - New Licensee Code>) must be considered instead.
 (The SGB will ignore any [command packets](<#Command Packet Transfers>) unless this value is `$33`.)
 
-Here is [a list of known Old Licensee Codes](https://raw.githubusercontent.com/gb-archive/salvage/master/txt-files/gbrom.txt).
+Here is a list of known Old Licensee Codes.
+
+HEX   | Licensee
+------|------------
+00    | None               
+01    | Nintendo           
+08    | Capcom
+09    | Hot-B              
+0A    | Jaleco             
+0B    | Coconuts
+0C    | Elite Systems      
+13    | EA (Electronic Arts)    
+18    | Hudsonsoft
+19    | ITC Entertainment  
+1A    | Yanoman            
+1D    | Clary
+1F    | Virgin Interactive             
+24    | PCM Complete       
+25    | San-X
+28    | Kotobuki Systems   
+29    | Seta               
+30    | Infogrames
+31    | Nintendo           
+32    | Bandai             
+33    | "see above"
```

Comment:
Should this be something like “use extended licensee code”?

Reply:
```suggestion
33    | Indicates that the [New Licensee Code](<#0144-0145 - New Licensee Code>) should be used instead.
```

Reply:
@ISSOtm do we have a particular reason to call it "New"? I kinda like "Extended" more

Reply:
Well, it's the "new" code because it superseded the "old" one. I'm fine with "extended" if you think it's better.

Reply:
I'll open an issue !